runtime/doc/vim2html.pl

```diff
@@ -1,16 +1,23 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
```

Comment:
that looks wrong

Reply:
As I understand it, this is not really required for Windows OS. On the contrary, it is required for UNIX-like systems. But judging by [this comment](https://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my), the attitude to "env" is not quite unambiguous.
I did it as it was originally.